inference-engine/ie_bridges/python/tests/test_InferRequest.py

```diff
@@ -525,17 +525,16 @@ def test_resize_algorithm_work(device):
     assert np.allclose(res_1, res_2, atol=1e-2, rtol=1e-2)
 
 
-# issue 56653
-@pytest.mark.skip(reason="Test will enable when nGraph Python API allows to create network with memory")
 def test_query_state(device):
     import ngraph as ng
     from ngraph.impl import Function
-    input_data = ng.parameter([5, 7], name="input_data", dtype=np.float32)
+
+    input_data = ng.parameter([10], name="input_data", dtype=np.float32)
```

Comment:
Does it make sense to have tests for other shapes and precisions?

Reply:
It makes prefect sense when we are looking at the overall test coverage and early catching of bugs. Parameterization of the test will do.

Reply:
parameterized